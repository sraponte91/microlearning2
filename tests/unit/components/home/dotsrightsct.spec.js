import { mount } from '@vue/test-utils'
import dotsrightsct from '@/components/home/dotsrightsct.vue'

const wrapper = mount(dotsrightsct)


describe('dotsrightsct.vue', () => {
    test('Is a Vue instance', () => {
        expect(wrapper.vm).toBeTruthy()
    })
    test('Renders correctly', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
})
