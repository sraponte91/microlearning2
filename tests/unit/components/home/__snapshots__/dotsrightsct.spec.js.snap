// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dotsrightsct.vue Renders correctly 1`] = `
<div
  class="dots-three-rt position-absolute"
>
  <div
    class="d-flex"
  >
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
  </div>
   
  <div
    class="d-flex"
  >
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
  </div>
   
  <div
    class="d-flex"
  >
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
  </div>
   
  <div
    class="d-flex"
  >
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
  </div>
   
  <div
    class="d-flex"
  >
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
  </div>
   
  <div
    class="d-flex"
  >
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
  </div>
   
  <div
    class="d-flex"
  >
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
  </div>
   
  <div
    class="d-flex"
  >
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
  </div>
   
  <div
    class="d-flex"
  >
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
  </div>
   
  <div
    class="d-flex"
  >
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
     
    <span />
  </div>
</div>
`;
